tests/testDipole.py

```diff
@@ -80,10 +80,10 @@ def createDipole(w, h, xc, yc, scaling=100.0, fracOffset=1.2):
     offset = fracOffset * psfFwhmPix // 2
     array = image.getImage().getArray()
     xp, yp = xc - psfw//2 + offset, yc - psfh//2 + offset
-    array[yp:yp+psfh, xp:xp+psfw] += psfim.getArray()
+    array[int(yp):int(yp)+psfh, int(xp):int(xp)+psfw] += psfim.getArray()
```

Comment:
I really like this solution. I find it very readable. Note that it does have the same effect as what I asked for in the first place: it is obvious that the index will be int, which is comforting to the reader but which @jdswinbank raised objections to.